min_distance_to_block: 1.5    
moving_random_distance: 0.5   
